geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaDataStore.scala

```diff
@@ -18,6 +18,7 @@ import org.geotools.data.DataAccessFactory.Param
 import org.geotools.data.store.{ContentDataStore, ContentEntry, ContentFeatureSource}
 import org.geotools.data.{DataStore, DataStoreFactorySpi, Query}
 import org.geotools.feature.NameImpl
+import org.opengis.feature.simple.SimpleFeatureType
```

Comment:
[minor] presumably left over imports that should be removed? Also can remove the extra space added below just to keep the history of the file cleaner.